setup.cfg

```diff
@@ -51,8 +53,13 @@ install_requires =
     numpy
     onnx==1.11.0
     onnxruntime==1.11.1
+    pyparsing
     sigtools==2.0.3
     toposort>=1.5.0
+    tf2onnx>=1.9.3
+    QKeras==0.9.0
+    tensorflow==2.7.0
```

Comment:
This is a quite heavy set of dependencies (especially tensorflow). It would be best to have this as an optional dependency as part of the `extras_require` section in setup.cfg, something like

```
[options.extras_require]
qkeras =
    tf2onnx>=1.9.3
    tensorflow==2.7.0
```
and so on.